### ISSUE TITLE
Migration Issue

### ISSUE THREAD

**Issue by jdvs1** [almightycouch/rethinkdb_ecto#34]
Hello again. I hope not to be annoying and continue bringing up issues here. I attempted to create a migration and it failed. Is that due to this driver or ignorance on my part? Here's the session:

```
$ # success in creating the db \o/ i then make adjustments to web/models/user.ex

$ mix ecto.create  # i meant to run migrate, oops
Compiling 1 file (.ex)
The database for MyApi.Repo has already been created

$ mix ecto.migrate
Compiling 1 file (.ex)

17:04:47.178 [info]  == Running MyApi.Repo.Migrations.CreateUser.change/0 forward

17:04:47.179 [info]  create table users

17:04:47.815 [info]  == Migrated in 0.6s
** (UndefinedFunctionError) function RethinkDB.Ecto.insert/6 is undefined or private. Did you mean one of:

      * insert/5

    (rethinkdb_ecto) RethinkDB.Ecto.insert(MyApi.Repo, %{autogenerate_id: nil, context: nil, schema: Ecto.Migration.SchemaMigration, source: {nil, "schema_migrations"}}, [version: 20170208223122, inserted_at: {{2017, 2, 9}, {0, 4, 47, 821879}}], {:raise, [], []}, [], [skip_transaction: true, timeout: :infinity, log: false])
    (ecto) lib/ecto/repo/schema.ex:459: Ecto.Repo.Schema.apply/4
    (ecto) lib/ecto/repo/schema.ex:198: anonymous fn/11 in Ecto.Repo.Schema.do_insert/4
    (ecto) lib/ecto/repo/schema.ex:128: Ecto.Repo.Schema.insert!/4
    (ecto) lib/ecto/migrator.ex:244: anonymous fn/4 in Ecto.Migrator.migrate/4
    (elixir) lib/enum.ex:1184: Enum."-map/2-lists^map/1-0-"/2
    (ecto) lib/mix/tasks/ecto.migrate.ex:84: anonymous fn/4 in Mix.Tasks.Ecto.Migrate.run/2
    (elixir) lib/enum.ex:651: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:651: Enum.each/2
    (mix) lib/mix/task.ex:296: Mix.Task.run_task/3
    (mix) lib/mix/cli.ex:58: Mix.CLI.run_task/2
    (elixir) lib/code.ex:363: Code.require_file/2
```

**Comment by redrabbit** [comment]
Hi @jdvs1, can you update to version [0.6.3](https://github.com/almightycouch/rethinkdb_ecto/releases/tag/v0.6.3) and check if the error still occurs? I think it has been fixed in the last version. Thank you in advance.

**Comment by redrabbit** [comment]
This issues has been fixed in version 0.6.3. Please reopen if you still have problems.
